app/shared/api-services/poll/answer-factory.tests.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+describe('Answer Factory', function () {
+  var $httpBackend,
+      answer,
+      answerFactory,
+      scope,
+      pollId;
+
+  beforeEach(module('apiServices.answer.factory'));
+
+  beforeEach(inject(function(_$httpBackend_, _Answer_) {
+    answerFactory = _Answer_;
```

Comment:
I've seen this before with other code. @kand / @MelizzaP out of curiosity, why do you need `_` around them, e.g. `_$httpBackend_` instead of `$httpBackend`.  Thought the latter worked fine too?